### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by bohdan-mykhailenko**

### CODE REVIEW

File: src/averageAges.js [hunk 1]
```diff
@@ -34,10 +49,32 @@ function calculateMenAverageAge(people, century) {
  * @param {object[]} people
  * @param {boolean} withChildren - optional
  *
- * @return {number}
+ * @return {Sum}
```
Comment (dinaminahi) [inline src/averageAges.js:52]: Change the type of return value from 'Sum' to 'number'.

File: src/averageAges.js [hunk 2]
```diff
@@ -10,16 +9,32 @@
  * Divide year of person's death by 100: Math.ceil(person.died / 100)
  *
  * @param {object[]} people
- * @param {number} century - optional
+ * @param {Sum} century - optional
  *
- * @return {number}
+ * @return {Sum}
```
Comment (dinaminahi) [inline src/averageAges.js:14]: Change the type of return value from 'Sum' to 'number'.

**Review by dinaminahi — CHANGES_REQUESTED** [review]
Good job! 👍 
Few moments to fix

**Review by naziamoff — APPROVED** [review]
👍

**Pull request closed without merging**